### PR TITLE
Add support for rails 5.0

### PR DESCRIPTION
**Pull request by saturnflyer**
Drops support for old Rubies and Rails. Adds support for newer (less than Rails 6)

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.1.5
+2.4
```
Comment (grosser) [inline .ruby-version]: needs to have a patch version

File: .travis.yml [hunk 2]
```diff
@@ -14,14 +18,13 @@ before_script:
   - mysql -u root -e 'CREATE DATABASE arsi_test CHARACTER SET utf8'
 
 gemfile:
- - gemfiles/rails3.gemfile
  - gemfiles/rails40.gemfile
  - gemfiles/rails41.gemfile
  - gemfiles/rails42.gemfile
```
Comment (grosser) [inline .travis.yml]: delete 40 / 41

File: .travis.yml [hunk 3]
```diff
@@ -14,14 +18,13 @@ before_script:
...
+ - gemfiles/rails50.gemfile
```
Comment (grosser) [inline .travis.yml]: might as well add 51/52 if it's not causing too much extra work

Comment (saturnflyer) [reply .travis.yml]: Will do. Wanted to settle this first and then add the files.

**Pull request merged**